src/Raven.Client/Documents/Subscriptions/SubscriptionCriteria.cs

```diff
@@ -113,8 +113,35 @@ public override void ConvertToJavascript(JavascriptConversionContext context)
                     }
 
                     return;
-                }                
-                
+                }
+
+                var newExp = context.Node as NewExpression;
+
+                if (newExp != null && newExp.Type == typeof(DateTime))
+                {
+                    context.PreventDefault();
+                    var writer = context.GetWriter();
+
+                    using (writer.Operation(newExp))
+                    {
+                        var args = newExp.Arguments;
+                        var argsStr = string.Empty;
+
+                        for (var i=0; i < args.Count; i++)
+                        {
+                            argsStr += args[i];
+
+                            if (i < args.Count - 1)
+                            {
+                                argsStr += ", ";
+                            }
+                        }
+
+                        writer.Write($"new Date({argsStr})");
```

Comment:
yes, as long as it's not one of the DateTime() overloads that receives a Calender and/or DateTimeKind arguments.